front/app/components/IdeaCards/IdeasWithFiltersSidebar/ContentRight.tsx

```diff
@@ -62,28 +62,14 @@ const ContentRight = ({
   onChangeTopics,
   onChangeSort,
 }: Props) => {
-  const [isCTABarVisible, setIsCTABarVisible] = useState(false);
-
-  useEffect(() => {
-    function checkCTABarVisibility() {
-      if (document.getElementById('project-cta-bar')) {
-        setIsCTABarVisible(true);
-        return;
-      }
-
-      setIsCTABarVisible(false);
-    }
-
-    window.addEventListener('scrollend', checkCTABarVisibility);
-    return () => window.removeEventListener('scrollend', checkCTABarVisibility);
-  }, []);
+  const stickyTop = document.getElementById('project-cta-bar-top');
 
```

Comment:
Just to confirm - is there any chance this line (65) could fire before the `project-cta-bar-top` appears in the document?

Reply:
@amanda-anderson Good point! Definitely, that chance exists. There are more reliable solutions (such as context) to determine the presence of the top CTA bar (same goes for bottom CTA bar in PlatformFooter). However, given a couple of factors, I will go with the solution as is:
- We have already spent quite some time on this. I think the code improvement is great, and will pay dividends in the future, but the product has improved minimally.
- The code is not crucial. If there's a very occasional bug, the only thing that will happen is that the filters will be pushed down a bit more on desktop, which is acceptable.

If we see that the bug occurs often in production, we can look into higher-effort solutions, perhaps.

I did add some comments to indicate the brittleness of the current solution.